src/Grid.jsx

```diff
@@ -64,8 +74,10 @@ function Grid({
                     ) : null}
                 </Fragment>
             ))}
-            {/*SVG of all the pattern geometrical shapes */}
-            {/* When clicking in the SVG, it creates a new point */}
+
+            {/*SVG of all the geometrical shapes of the pattern */}
+            {/*When clicking in the SVG, it creates a new point */}
+            {/*The mouse position in the grid is always recorded for drag'n'drops */}
             <svg
                 ref={SVGRef}
                 width={width}
```

Comment:
issue (about the `onClick`, line 89): this function should be defined in the component rather than directluy in the JSX. Also you should probably have some helper function for the computation that begins with `parseFloat`.